src/main/java/org/vietabroader/model/VASpreadsheet.java

```diff
@@ -15,8 +17,61 @@
  */
 public class VASpreadsheet {
 
+
     private String spreadsheetId;
     private Spreadsheet spreadSheet;
+    private HashMap<String,String> columnMap;
```

Comment:
Other than columns, we still need 2 more variables to hold the starting row and ending row
```
private int fromRow;
private int toRow;
```